app/views/shops/orders.html.erb

```diff
@@ -14,8 +14,8 @@
   <div class="col s5 pagination-holder">
     <ul class="tabs no-border-bottom">
       <li class="tab col s3"><a href="" >All (<%= all_orders_count %>)</a></li>
-      <li class="tab col s3"><a  href="?status=completed">Completed (<%= Order.pending(@shop).count %>)</a></li>
-      <li class="tab col s3"><a href="?status=pending" >Pending (<%= Order.completed(@shop).count %>)</a></li>
+      <li class="tab col s3"><a  href="?status=completed">Completed (<%= Order.completed(@shop).count %>)</a></li>
```

Comment:
If I'm right with what you are doing here, then we can have the completed method in the Shop model